### DISCUSSION TITLE
Grapes JS CSS editor

### DISCUSSION THREAD

**Discussion by mdshimul186** [GrapesJS/grapesjs#5564] [Q&A]
In grape js demo i have seen a custom css editor from where i can write css and update the style of selected component. How to do that in my project.  I have installed the grapes JS in my project.

**Comment by ronaldohoch** [comment]
I think this is a plugin.
Maybe this: https://github.com/GrapesJS/components-custom-code
Or that import button: https://github.com/GrapesJS/preset-webpage/blob/master/src/commands/openImport.ts
